src/wallet/ismine.cpp

```diff
@@ -162,6 +162,11 @@ IsMineResult IsMineInner(const CWallet& keystore, const CScript& scriptPubKey, I
         }
         break;
     }
+    case TX_SECURETHEBAG:
+    {
+        ret = IsMineResult::NO;
```

Comment:
conceptual question: where will this be handled?

Reply:
yah, I'm familiar with the function, I mean do we anticipate nodes will store known Bag info at all? or is that going to be handled solely by external software?

Reply:
Yes, if you want to receive a payment, you should store the TX_SECURETHEBAG data in your wallet.